packages/sync-actions/src/product-actions.js

```diff
@@ -635,6 +636,57 @@ export function actionsMapPrices(diff, oldObj, newObj, variantHashMap) {
   return changePriceActions.concat(removePriceActions).concat(addPriceActions)
 }
 
+export function actionsMapPricesCustom(diff, oldObj, newObj, variantHashMap) {
+  let actions = []
+
+  const { variants } = diff
+
+  if (variants)
+    forEach(variants, (variant, key) => {
+      const { oldObj: oldVariant, newObj: newVariant } = extractMatchingPairs(
+        variantHashMap,
+        key,
+        oldObj.variants,
+        newObj.variants
+      )
+
+      if (
+        variant &&
+        variant.prices &&
+        (REGEX_UNDERSCORE_NUMBER.test(key) || REGEX_NUMBER.test(key))
+      ) {
+        const priceHashMap = findMatchingPairs(
+          variant.prices,
+          oldVariant.prices,
+          newVariant.prices
+        )
+
+        forEach(variant.prices, (price, index) => {
+          const { oldObj: oldPrice, newObj: newPrice } = extractMatchingPairs(
+            priceHashMap,
+            index,
+            oldVariant.prices,
+            newVariant.prices
+          )
+
+          if (
+            price.custom &&
+            (REGEX_UNDERSCORE_NUMBER.test(index) || REGEX_NUMBER.test(index))
+          ) {
+            const a = actionsMapCustom(price, newPrice, oldPrice, {
```

Comment:
Can we change the variable name `a` here? I believe something descriptive will be better.

Reply:
Done 👍